spec.html

```diff
@@ -27240,7 +27236,7 @@ <h1>Number.prototype.toExponential ( _fractionDigits_ )</h1>
           1. Return the string-concatenation of _s_ and _m_.
         </emu-alg>
         <emu-note>
-          <p>For implementations that provide more accurate conversions than required by the rules above, it is recommended that the following alternative version of step 10.b.i be used as a guideline:</p>
+          <p>For implementations that provide more accurate conversions than required by the rules above, it is recommended that the following alternative version of step 9.b.i be used as a guideline:</p>
```

Comment:
Note to self: https://github.com/tc39/ecmarkup/issues/192 needs addressing so this can't happen.

Reply:
See also: <https://github.com/tc39/ecmarkdown/issues/56>